src/FirebaseArray.js

```diff
@@ -540,7 +551,7 @@
        * @param {Object} [methods] a list of functions to add onto the prototype
```

Comment:
I'm updating these now, along with the short comments at the top you mentioned earlier.